src/transports/types.ts

```diff
@@ -240,11 +264,52 @@ export interface IImageTrackSegmentData {
   type : string; // the type of the data (example: "bif")
 }
 
+// Response from a video segment parser when parsing an init segment
```

Comment:
I don't see the interset of creating both video aud audio types here.

Reply:
done